authors/apps/articles/tests/test_data.py

```diff
@@ -6,3 +6,5 @@
         "body": "this is the body"
     }
 }
+
+
```

Comment:
Hello Roy could you keep just one blank at the end of this code as per the pep8 guidelines